src/js/select2/a11y/a11yMulti.js

```diff
@@ -0,0 +1,63 @@
+define([
+  '../utils'
+], function (Utils) {
+  function A11yMulti (decorated, $element, options) {
+    decorated.call(this, $element, options);
+  }
+
+  A11yMulti.prototype.bind = function (decorated, container, $container) {
+
+    // Add a container for accessible selection summary
+    var selectionSummaryId = container.id + '-summary';
+    this.$selectionSummary = $('<span id="'+ selectionSummaryId +'" class="select2-selections"></span>');
+    $container.append(this.$selectionSummary);
+
+    // If orginal select had aria-describedby, add to select2 search
+    if (this.$element.attr('aria-describedby') != null) {
```

Comment:
`attr(...)` (https://api.jquery.com/attr/#attr-attributeName) returns `undefined` when an attr is not present, should this be checking if `!== undefined` ?